interfaces/seccomp/backend.go

```diff
@@ -143,25 +150,32 @@ func (b *Backend) Remove(snapName string) error {
 // deriveContent combines security snippets collected from all the interfaces
 // affecting a given snap into a content map applicable to EnsureDirState.
 func (b *Backend) deriveContent(spec *Specification, opts interfaces.ConfinementOptions, snapInfo *snap.Info) (content map[string]*osutil.FileState, err error) {
+	addSocketcall := false
+	// Some base snaps and systems require the socketcall() in the default
+	// template
+	if requiresSocketcall(snapInfo.Base) {
```

Comment:
Sorry, I think I missed that earlier. Could we simply write: `addSocketcal := requiresSocketcall(snapInfo.Base)` here?

Reply:
Yeah, that's fine. It made more sense in an earlier iteration.